src/App.tsx

```diff
@@ -1,14 +1,27 @@
-import { FooterComponent } from '@/components/footer/Footer.component.tsx';
-import { HeaderComponent } from '@/components/header/Header.component.tsx';
+import { useState } from 'react';
 
-import { AboutComponent } from './components/about/About.component.tsx';
+import AboutComponent from '@/components/about/About.component';
+import FooterComponent from '@/components/footer/Footer.component';
+import HeaderComponent from '@/components/header/Header.component';
+import ProductList from '@/components/product/ProductList.component';
+
+import styles from './App.module.css';
 
 function App() {
+    const [isShowAbout, setIsShowAbout] = useState(true);
+
+    const toggleAbout = () => {
+        setIsShowAbout(true);
+    };
+
+    const toggleProducts = () => {
+        setIsShowAbout(false);
+    };
```

Comment:
You can combine these two similar functions to one

Reply:
fixed